docs/src/pages/flags.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Feature Flags
+permalink: /flags/index.html
+eleventyNavigation:
+    key: feature flags
+    parent: use eslint
+    title: Feature Flags
+    order: 6
+---
+
+ESLint ships experimental and future breaking changes behind feature flags to let users opt-in to behavior they want. Flags are used in these situations:
+
+1. When a feature is experimental and not ready to be enabled for everyone.
+1. When a feature is a breaking change that will be formally merged in the next major release, but users may opt-in to that behavior prior to the next major release.
```

Comment:
```suggestion
2. When a feature is a breaking change that will be formally merged in the next major release, but users may opt-in to that behavior prior to the next major release.
```

Reply:
IG this will get fixed automatically by markdown when rendered

Reply:
The number doesn't matter. Markdown allows any number at the start. I always use 1 because then I never need to think about updating numbers if I move things around.